pkg/kv/kvserver/rangefeed/buffered_registration.go

```diff
@@ -84,6 +85,7 @@ func newBufferedRegistration(
 ) *bufferedRegistration {
 	br := &bufferedRegistration{
 		baseRegistration: baseRegistration{
+			streamCtx:        streamCtx,
```

Comment:
After the legacy process removal, I think we can also get rid of this stored context because the ScheduledProcessor calls runOutputLoop as part of the call to Register() so we will have the right context being passed into us.

Reply:
I will address this in a follow up PR after we remove legacy processor.